crates/nargo/tests/test_data/pedersen_check/Prover.toml

```diff
@@ -2,5 +2,5 @@ x = "0"
 y = "1"
 salt = "42"
 
-out_x = "0x108800e84e0f1dafb9fdf2e4b5b311fd59b8b08eaf899634c59cc985b490234b"
-out_y = "0x2d43ef68df82e0adf74fed92b1bc950670b9806afcfbcda08bb5baa6497bdf14"
+out_x = "0x229fb88be21cec523e9223a21324f2e305aea8bff9cdbcb3d0c6bba384666ea1"
```

Comment:
Seems like quite a few of the Prover files have changed. Does hashing depend on our backend version?

Reply:
Our pedersen does rely on our backend version. With the switch to `noir-lang/aztec-connect` pedersen changed such that new outputs had to be reflected.

Reply:
> I had to update some inputs to the nargo functions to reflect changes in the functionality of std_lib functions from barretenberg. These included different signatures for programs that use schnorr and changes to hash outputs for anything that uses pedersen.

Same thing with schnorr